sumologic/resource_sumologic_data_forwarding_rule.go

```diff
@@ -0,0 +1,167 @@
+package sumologic
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"log"
+)
+
+func resourceSumologicDataForwardingRule() *schema.Resource {
+	return &schema.Resource{
+
+		Create: resourceSumologicDataForwardingRuleCreate,
+		Read:   resourceSumologicDataForwardingRuleRead,
+		Update: resourceSumologicDataForwardingRuleUpdate,
+		Delete: resourceSumologicDataForwardingRuleDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			"index_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(16, 16),
+			},
+			"destination_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(16, 16),
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"file_prefix": {
```

Comment:
any particular reason we are using file_prefix instead of file_format, since in documentation also file_format is used right ?

Reply:
reason is I was thinking having same name will help in reading

Reply:
Nice pick, I thought of keeping it "file_prefix" just for readability .. but I've refactored it back to "file_format" in my recent commit.